### PR TITLE
Update communication page

### PR DESCRIPTION
**Pull request by manuelakuhn**
Added documentation for the mattermost instance available from the computer science department and refer to the e-mail configuration page from the URZ directly to avoid mistakes.

### CODE REVIEW

File: content/pages/services/communication.rst [hunk 1]
```diff
@@ -9,26 +9,8 @@ OvGU Email
...
-  - security: SSL/TLS
-  - authentication method: password, normal
-
-* SMTP
-
-  - server: ``mail.ovgu.de``
-  - port: 587
-  - security: STARTTLS (TLSv1)
-  - authentication method: password, normal
+* via web client of your choice. A detailed description on the configuration can be found `here <https://www.urz.ovgu.de/Unsere+Leistungen/Anwendungen/E_Mail/Konfiguration.html>`_
```
Comment (aqw) [inline content/pages/services/communication.rst]: "via web client of your choice" seems out of place, as this line is about email clients, and the previous line is about webmail.

**Review by aqw — COMMENTED** [review]
Is there a motivation for removing this information?

File: content/pages/services/communication.rst (hunk 1 above)
Comment (manuelakuhn) [reply content/pages/services/communication.rst]: Yes, you are right. Web client is wrong here and it should state email client. Sorry, I have fixed this.

**Comment by manuelakuhn** [comment]
The motivation was to avoid mistakes while duplicating information already present in the official URZ documentation. Since there was already a typo present in the server name, the options were to either fix it or to link to the official documentation. Doing the latter seemed more reasonable to me.

**Comment by aqw** [comment]
Makes sense. :-)

In the past, the URZ pages were not always reliable, but that doesn't seem to be a problem here.

Thanks for the PR. :-)

**Comment by manuelakuhn** [comment]
Thanks :-)

**Pull request merged**